### PR TITLE
chore(package): dependency updates

### PR DESCRIPTION
**Pull request by alexlafroscia**

### CODE REVIEW

File: tests/helpers/start-app.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import Ember from 'ember';
 import Application from '../../app';
 import config from '../../config/environment';
 
-const { merge, run } = Ember;
+const { assign, merge, run } = Ember;
```
Comment (jherdman) [inline tests/helpers/start-app.js]: merge appears to be unused and is also deprecated.

File: package.json [hunk 2]
```diff
@@ -25,17 +28,20 @@
...
   "license": "MIT",
+  "dependencies": {
+    "ember-cli-babel": "^5.1.5"
+  },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.3",
-    "coveralls": "^2.11.13",
-    "ember-cli": "2.9.1",
+    "coveralls": "^2.11.15",
+    "ember-cli": "^2.10.0",
```
Comment (jherdman) [inline package.json]: Huh, weird. I didn't think ember-cli was a floating version after running the upgrade procedure. Is this safe?

File: package.json [hunk 3]
```diff
@@ -2,6 +2,9 @@
   "name": "ember-ajax",
   "version": "2.5.2",
   "description": "Service for making AJAX requests in Ember 1.13+ applications.",
+  "keywords": [
+    "ember-addon"
```
Comment (jherdman) [inline package.json]: Should we consider adding "ajax" to the keyword list whilst we're at it?

File: package.json (hunk 2 above)
Comment (alexlafroscia) [inline package.json]: Hmm, interesting. I do the dependency updates for `ember-ajax` through Yarn, and that is what it changed it to by default.

Comment (jherdman) [inline package.json]: Very interesting. It makes me a little uncomfortable allowing flexibility on the minor bit given that ember-cli has an involved upgrade process. Perhaps we should consider nailing this down to an explicit version as per the usual npm-based upgrade process?

Comment (alexlafroscia) [inline package.json]: Sure, I'll re-pin it to the exact version. 

This really highlights how the CLI doesn't quite fit with Semver. A minor version *should not require changes* but always does.

**Pull request closed without merging**